sources/Interop/WinTrust/um/wintrust/CRYPT_PROVIDER_CERT.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/WinTrust.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct CRYPT_PROVIDER_CERT
+    {
+        [NativeTypeName("DWORD")]
+        public uint cbStruct;
+
+        [NativeTypeName("PCCERT_CONTEXT")]
+        public _CERT_CONTEXT* pCert;
```

Comment:
Should be remapped and defined also from wincrypt.

Reply:
Same with `_CTL_CONTEXT` and `_CERT_CHAIN_ELEMENT` below

Reply:
Fixed.